tests/contrib/django/test_django_appsec.py

```diff
@@ -12,3 +12,19 @@ def test_django_simple_attack(client, test_spans, tracer):
     assert "triggers" in json.loads(root_span.get_tag("_dd.appsec.json"))
     assert _context.get_item("http.request.uri", span=root_span) == "http://testserver/.git?q=1"
     assert _context.get_item("http.request.headers", span=root_span) is not None
+    assert _context.get_item("http.request.query", span=root_span)["q"] == "1"
+
+
+def test_django_querystrings(client, test_spans, tracer):
+    tracer._appsec_enabled = True
+    # Hack: need to pass an argument to configure so that the processors are recreated
+    tracer.configure(api_version="v0.4")
+    client.get("/?a=1&b&c=d")
+    root_span = test_spans.spans[0]
+    assert _context.get_item("http.request.query", span=root_span)["a"] == "1"
+    assert _context.get_item("http.request.query", span=root_span)["b"] == ""
+    assert _context.get_item("http.request.query", span=root_span)["c"] == "d"
+    assert "d" not in _context.get_item("http.request.query", span=root_span)
+    client.client.get("/")
+    root_span = test_spans.spans[0]
+    assert len(_context.get_item("http.request.query", span=root_span)) == 0
```

Comment:
```suggestion
    assert not _context.get_item("http.request.query", span=root_span)
```

```suggestion
    assert _context.get_item("http.request.query", span=root_span) == {}
```

(what you have is 100% fine, just showing other Python ways to assert this, no change needed)